### PR TITLE
Add ability to deploy automatically with serverless and plugins

### PR DESCRIPTION
**Pull request by rss-maksim**
Task: https://github.com/rolling-scopes-school/nodejs-aws-tasks/blob/main/task2-serve-spa-aws/task.md

All points have been done:
- manual deployment
- automated deployment using serverless-finch + serverless single-page-app plugins 

**Manual Build**:
**S3 bucket link (Forbidden)**: http://manual-bucket-application.s3-website-eu-west-1.amazonaws.com
**CloudFront website link:** https://d3k5jlawahu2sy.cloudfront.net

**CloudFormation Build**:

**S3 bucket link(Forbidden)**: http://kirilko-app2.s3-website-us-east-1.amazonaws.com
**CloudFront website link:** https://d2kxmdanbdhxtf.cloudfront.net

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,34 +5,48 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "build:deploy": "npm run build && npm run deploy:s3",
```
Comment (aosin) [inline package.json]: You probably forgot to remove this line after adding "client:build:deploy"

File: package.json [hunk 2]
```diff
@@ -5,34 +5,48 @@
...
+    "client:deploy": "serverless client deploy --region eu-west-1 --no-config-change --no-policy-change --no-cors-change",
```
Comment (aosin) [inline package.json]: You can put region in `serverless.yml` under "provider" section.

File: serverless.yml [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+plugins:
+  - serverless-finch
+  - serverless-single-page-app-plugin
+
+custom:
+  client:
+    bucketName: kirilko-app2
+    distributionFolder: build
+    indexDocument: index.html
+    manageResources: false
```
Comment (aosin) [inline serverless.yml:38]: If you set `manageResources` to `false` here, then there is no need to add `--no-config-change --no-policy-change --no-cors-change` to the command line (serverless-finch "manages" your resources by default, this is what you are opting out of by adding those switches)

**Review by aosin — COMMENTED** [review]
It is OK to merge, but you may wish to check out my comments first.

File: serverless.yml (hunk 3 above)
Comment (rss-maksim) [reply serverless.yml:38]: Thanks

**Pull request merged**